### PR TITLE
Removes output from Examples that break the docs

### PR DESCRIPTION
**Pull request by MikeyBronowski**
## Type of Change

 - [ ] Bug fix (non-breaking change, fixes # )
 - [ ] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1`)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Unit test is included
 - [x] Documentation
 - [ ] Build system

### Purpose

The docs are broken by this format, plus other examples do not include outputs.

image

### CODE REVIEW

File: functions/Add-DbaAgListener.ps1 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 function Add-DbaAgListener {
     <#
-    .SYNOPSIS
+        .SYNOPSIS
```
[the remark this answers is not in the document]
Comment (MikeyBronowski) [reply functions/Add-DbaAgListener.ps1]: Yeah, i was looking for indentation so had to get extra tab :/ thanks for spotting

**Review by andreasjordan — APPROVED** [review]
I'm totally fine with removing the output.

**Comment by niphlod** [comment]
all it takes is prefixing the output with > to not break the docs

**Comment by potatoqualitee** [comment]
Thank you, @niphlod . @MikeyBronowski would you be able to prefix the output with `> ` instead? 🙏🏼

**Comment by MikeyBronowski** [comment]
@potatoqualitee @niphlod done

**Review by wsmelton — APPROVED** [review]

**Comment by potatoqualitee** [comment]
awessome thank you all!

**Pull request merged**